api.md

```diff
@@ -1036,6 +1037,60 @@ CALL distributed_exec('CREATE DATABASE dist_database', transactional => FALSE);
 
 ---
 
+## create_distributed_restore_point() [](create_distributed_restore_point)
+
+Create a named point for performing restore across a distributed database.
```

Comment:
Consistency, do we say "distributed database" or "multi-node TimescaleDB cluster"?